25.md

```diff
@@ -7,20 +7,16 @@ Reactions
 
 `draft` `optional`
 
-A reaction is a `kind 7` event that is used to react to other events.
+A reaction is a `kind 7` event that is used to indicate user reactions to other events. A
+reaction's `content` field MUST include a single character indicating the value of the reaction.
```

Comment:
Custom reactions `:name:` are not single chars. Also, many emojis are not single chars. It's common to see emoji blends, which depending on the OS's support, renders as one or more chars.

Reply:
Got it, will update that